File: Beecrowd/8-Geometria_Computacional/1039-Flores_de_Fogo-5/1039.cs
using System;

class Solution {
    static void Main() {
        string input;
        while ((input = Console.ReadLine()) != null) {
            string[] values = input.Split();
            double r1 = double.Parse(values[0]);
            double x1 = double.Parse(values[1]);
            double y1 = double.Parse(values[2]);
            double r2 = double.Parse(values[3]);
            double x2 = double.Parse(values[4]);
            double y2 = double.Parse(values[5]);

            double distance = Math.Sqrt((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2));

            Console.WriteLine(distance + r2 <= r1 ? "RICO" : "MORTO");
        }
    }
}

File: Beecrowd/8-Geometria_Computacional/1039-Flores_de_Fogo-5/1039.py
import sys
import math

for line in sys.stdin:
    r1, x1, y1, r2, x2, y2 = map(float, line.split())
    distance = math.sqrt((x1 - x2) ** 2 + (y1 - y2) ** 2)
    print("RICO" if distance + r2 <= r1 else "MORTO")
